frontend/tasks/scripts.js

```diff
@@ -55,36 +61,72 @@ gulp.task('scripts-misc', () => {
 });
 
 gulp.task('scripts-generate-static-html', () => {
-  return commonBrowserify('generate-static-html.js', browserify({
+  return browserify({
```

Comment:
Might be better to tweak `commonBrowserify` to accept an option for output path than copypasta most of it here.